ts-serverless-quickstart/README.md

```diff
@@ -0,0 +1,26 @@
+# Serverless Express Example
+
+The purpose of this project is to showcase how you would update an existing Klotho-annotated application to include new services and routes. 
+
```

Comment:
Given that this sample app isn't really a "standalone" as the others is, but is really meant for the tutorial, I might add something like:

```suggestion

This project is intended to be used along with our [serverless quickstart tutorial](https://klo.dev/docs-v1/tutorials/serverless_quickstart).

```

I would especially add that if you want to keep the `fish.ts`, since otherwise it's kinda weird to have a file with just a comment saying "we'll fill it in later". (Reading that without context, I'd assume "we" is Klotho and that this is some sort of unfinished TODO.)

Reply:
yeh good shout, was gonna do that after the tutorial was merged in but can just do that now